.github/actions/build-node/action.yml

```diff
@@ -5,11 +5,6 @@ inputs:
   node-version:
     description: 'The version of Node.js to use'
     required: true
-  cache-built:
```

Comment:
This was needed for the deployment test job which couldn't use the build cache because it runs on a different linux environment, so the native modules would not match. The default value of `true` should have made it no impact on any other job. I need to understand why you claim it affected other jobs.

Reply:
I'll create a new PR to distinguish the cache key without an explicit toggle.